projects/1.19/assets/tanukidecor/tanukidecor/lang/zh_cn.json

```
@@ -0,0 +1,245 @@
+{
+  "itemGroup.tanukidecor": "狸猫之家",
+
+  "block.tanukidecor.alarm_clock": "闹钟",
+  "block.tanukidecor.anniversary_clock": "周年纪念钟",
+  "block.tanukidecor.antique_clock": "古董钟",

```

```suggestion
  "block.tanukidecor.antique_clock": "古董时钟",
```
建议尽量采用《集合吧》版本的官中。《集合吧》没有的情况下（例如前作家具或原创物品）再去考虑其他翻译
https://nookea.com/zh_cn/ac/54WzSWz3bv4j4279w